cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -281,30 +297,45 @@ export default {
     cy.wait(1000);
   },
 
-  addNote:(note) => {
+  addAdministrativeNote:(note) => {
     cy.do(Select({ name:'profile.mappingDetails.mappingFields[9].repeatableFieldAction' }).choose('Add these to existing'));
     cy.do(Button('Add administrative note').click());
     cy.do(TextField('Administrative note').fillIn(note));
   },
 
-  // fill fields of instance mapping profile
+  addName:(name) => {
+    cy.do(TextField({ name:'profile.name' }).fillIn(name));
+  },
+
+  addIncomingRecordType:(type) => {
+    cy.do(Select({ name:'profile.incomingRecordType' }).choose(type));
+  },
+
+  addFolioRecordType:(folioType) => {
+    cy.do(Select({ name:'profile.existingRecordType' }).choose(folioType));
+  },
+
+  saveProfile:() => {
+    cy.wait(1000);
```

Comment:
add comment for waiter please. Are you sure that you can not use another way for waiting?

Reply:
I deleted it